src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -456,116 +484,128 @@ private async Task AcceptStreamsAsync()
         private async Task ProcessServerStreamAsync(QuicStream stream)
         {
             ArrayBuffer buffer = default;
+            QuicStream? quicStream = stream;
```

Comment:
In the original code we had `await using (stream.ConfigureAwait(false))` which disposed every stream from the server after it being processed, now since the webtransport streams cannot be disposed afterwards I added this variable instead so I can still dispose the other streams

Reply:
Interesting. Let me take a look at the original code.
Control streams should not be closed as well, so I need to check how they were preserved from being disposed.

But if we were to stick with your current approach, maybe we could use a bool `disposeStream` set to true and then set it to false when we decide we don't want to dispose in a specific case. This way it will be more clear what is happening and why.

Reply:
@CarnaViire because the `ProcessServerControlStreamAsync` function keeps reading the control stream inside a `while true` so it does not return and the stream does not get disposed in this function

Reply:
What do you think about bool value approach?

Reply:
Will implement it